syftbox/client/routers/sync_router.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+
+from fastapi import APIRouter, HTTPException
+from loguru import logger
+
+from syftbox.client.plugins.sync.local_state import LocalState, SyncStatusInfo
+from syftbox.client.routers.common import APIContext
+
+router = APIRouter()
+
+
+@router.get("/data")
```

Comment:
Discussed this with @yashgorana , we made some changes so we can read SyncManager state directly (instead of from local_state file). Next pr